src/App.js

```diff
@@ -16,16 +18,19 @@ function App() {
         <Routes>
 
           <Route path='/' element={<LandingPage />} />
-          <Route path='/sign-up' element={<Signup />} />
-          <Route path='/sign-in' element={<Signin />} />
-          <Route path='/property' element={<Property />} />
-          <Route path='/propertydetails' element={<PropertyDetails />} />
-            <Route path='/dashboard' element={<Dashboard/>} />
-            <Route path ='/propertypost' element={<Propertypost />} />
+          <Route path='/signup' element={<Signup />} />
+          <Route path='/signin' element={<Signin />} />
+          <Route path='/property' element={<Property/>} />
+          <Route path ='/edit/Property/:id' element={<Editproperty/>} />
+          <Route path='/allproperties' element={<Allproperty/>} />
```

Comment:
This is not a good name for a route, you should have something like `/properties`

Reply:
This is not resolved.